web/src/main/webapp/loc/ara/xml/strings.xml

```diff
@@ -1646,4 +1646,6 @@
   <tag_label>Tags</tag_label>
   <map_label>Map</map_label>
   <unknown>Unknown</unknown>
+
```

Comment:
I guess these files are not managed on Transifex, right?

Reply:
Afaik only the AngularJs UI files are managed in Transifex, not these ones.